gem/templates/springster/core/main.html

```diff
@@ -10,7 +10,7 @@
     {% media_listing_homepage %}
   {% else %}
     {% get_tag_articles tag_count=4 as tag_nav_data %}
-    {% paginate 12 tag_nav_data.latest_articles as latest_articles %}
+    {% paginate 15 tag_nav_data.latest_articles as latest_articles %}
```

Comment:
I wonder what the homepage loading impact will be with less articles. We were already pushing it with 12, now we are on 15, the home page is getting quite big, we should really rework this after our optimisation investigations.